### PR TITLE
docs: Add CODEOWNERS and contributing guide

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Tests are located in `tests` folder and run by [jest](https://jestjs.io/) in node environment.
+
+To run tests you can use IDE instruments or just run:
+```shell
+yarn test
+```
+
+### How to publish
+
+Change version in package.json to x.y.z and push a commit with the message Release x.y.z, the npm-publish action will create a new tag vx.y.z and publish the package to the npm registry.
```
Comment (ilfa) [inline contributing.md:37]: Check [GitHub action](https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk/blob/main/.github/workflows/npm-publish.yml
) please.

It triggers only by `^Release (\\S+)` pattern. We should say about it here. Or we can use `generate-changelog` package as [in some other repositories](https://github.com/fingerprintjs/fingerprintjs-pro-spa/blob/main/package.json#L26).

Comment (TheUnderScorer) [reply contributing.md]: Good point, I've done some further changes to use `generate-changelog` like in other repositories!

File: package.json [hunk 2]
```diff
@@ -11,7 +11,10 @@
     "build": "tsc",
     "lint": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 .",
     "lint:fix": "yarn lint --fix",
-    "test": "npx jest"
+    "test": "npx jest",
+    "release:major": "changelog -M && git add CHANGELOG.md && yarn version --major",
```
Comment (ilfa) [inline package.json:15]: As `npm-publish-action` creates tag we don't need it to create it in this step. Please add [--no-git-tag-version](https://classic.yarnpkg.com/en/docs/cli/version#toc-yarn-version-no-git-tag-version) for `yarn version` call.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#### 0.2.1 (2022-04-11)
+
+##### Chores
```
Comment (ilfa) [inline CHANGELOG.md:3]: Al the list looks like mostly a spoiler for the next version =)

**Review by ilfa — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (TheUnderScorer) [reply package.json]: Done!

File: package.json [hunk 4]
```diff
@@ -30,6 +33,7 @@
     "typescript": "^4.5.5"
   },
   "dependencies": {
+    "generate-changelog": "^1.8.0",
```
Comment (ilfa) [inline package.json:36]: Please move it to `devDependencies`

**Review by ilfa — CHANGES_REQUESTED** [review]

File: package.json (hunk 4 above)
Comment (TheUnderScorer) [reply package.json]: Done! :)

**Review by ilfa — APPROVED** [review]

**Pull request merged**